lib/checks/tables/th-has-data-cells.js

```diff
@@ -30,7 +30,10 @@ var tableGrid = tableUtils.toGrid(node);
 
 // Look for all the bad headers
 var out = headers.reduce(function (res, header) {
-	if (header.id && reffedHeaders.indexOf(header.id) !== -1) {
+	if (
```

Comment:
Does this code fit on one line under 80 characters? It would be more consistent with our existing style.